diff --git a/dist/fs.js b/dist/fs.js
index d8a6d7087959623419a0de66f35f20a18f6f6311..008f1edddfe2f0dfd3a3fd46be9bbf11f863fefd 100644
--- a/dist/fs.js
+++ b/dist/fs.js
@@ -46,10 +46,12 @@ async function copyFilesToFolder(files, outDir, exclude = []) {
       await fs.mkdir(new URL(".", dest), { recursive: true });
     }
     if (isSymlink) {
-      const realdest = fileURLToPath(new URL(nodePath.relative(commonAncestor, realpath), outDir));
-      const target = nodePath.relative(fileURLToPath(new URL(".", dest)), realdest);
       if (!existsSync(dest)) {
-        await fs.symlink(target, dest, isDir ? "dir" : "file");
+        if (isDir) {
+          await fs.cp(realpath, dest, { recursive: true });
+        } else {
+          await fs.copyFile(realpath, dest);
+        }
       }
     } else if (!isDir) {
       await fs.copyFile(origin, dest);
